recipe_scrapers/spisbedre.py

```diff
@@ -141,3 +157,29 @@ def cuisine(self):
 
     def description(self):
         return self.schema.description()
+
+    def equipment(self):
+        result = [
+            equipment.get("name")
+            for group in self.recipe_json.get("grouped_equipment", [])
+            for equipment in group.get("equipment", [])
+            if equipment.get("name")
+        ]
+        return get_equipment(result)
+
+    def nutrients(self):
+        nutrition = self.recipe_json.get("nutrition")
+        if not nutrition:
+            return None
+
+        calories = round(nutrition.get("calories"))
+        fat = round(nutrition.get("fat"))
+        carbohydrates = round(nutrition.get("carbohydrates"))
+        protein = round(nutrition.get("protein"))
```

Comment:
```suggestion
        def format_nutrient(value, unit):
            return f"{round(value)} {unit}"
```

Rather than using the same logic repeatably we can use a helper method here directly on the output.

Reply:
Great idea, I've implemented the necessary changes.